.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,6 +26,9 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
+**Logs:**
+ - Please attach logs for Bridge to Kubernetes to triage issues faster, here are the log locations ( Window C:\Users\Username\AppData\Local\Temp\Bridge To Kubernetes, Linux/Mac - $HOME/tmp/Bridge To Kubernetes or /tmp/Bridge To Kubernetes). Please attach logs relevant to your issue or bug. 
```

Comment:
☕️❤️👍 minor- for clarity add “windows -“ like “ Linux/Mac - ” - rest looks cool. Thank you 🙏